packages/fxa-payments-server/src/lib/apiClient.ts

```diff
@@ -1,5 +1,12 @@
 import { Config, defaultConfig } from './config';
-import { Plan, Profile, Customer, Subscription, Token } from '../store/types';
+import {
+  Plan,
+  Profile,
+  Customer,
+  Subscription,
+  Token,
+  FilteredSetupIntent,
```

Comment:
Might want to house this type here with the apiClient stuff. Those other types are for the Redux store, which should someday go away / [move out into fxa-shared](https://github.com/mozilla/fxa/issues/5912)